org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/internal/ui/preferences/XMLCatalogs.java

```diff
@@ -126,10 +126,17 @@ private static URI createURI(IConfigurationElement element) {
 		URI uri = URI.create(element.getAttribute("uri"));
 		if (!uri.isAbsolute()) {
 			try {
-				URL url = FileLocator.find(Platform.getBundle(element.getContributor().getName()),
+				String contributorName = element.getContributor().getName();
+				URL url = FileLocator.find(Platform.getBundle(contributorName),
 						Path.fromPortableString(uri.toString()));
-				// this constructor will ensure parts are URI encoded correctly
-				uri = new URI(url.getProtocol(), url.getAuthority(), url.getPath(), null, null);
+				if(Objects.nonNull(url)) {
```

Comment:
As `Objects` is not imported, it causes a compilation issue. Why not just using `url != null` here?
```suggestion
				if( (url != null) {
```

Reply:
I modified the code in the github editor, because my hard drive on which I fixed the code crashed. I forgot to add the import of Objects. My bad.
Personally I prefer the Objects.nonNull because it's less error prone than != and ==. You make errors with those operators more easily.